src/scss/_vars.scss

```diff
@@ -104,7 +104,7 @@ $gutter-size: 20px;
 // Responsive Breakpoints
 // key order matters (must be from smallest to largest breakpoint)
 $screen: (
-  xs: 0em,
```

Comment:
removed unnecessary `em` unit (`0em` is the same as `0px` which is the same as `0`)